lib/utils/testem-events.js

```diff
@@ -38,8 +38,8 @@ class TestemEvents {
     if (browserIdsToReplay && browserIdsToReplay.length > 0) {
       browserIdsToReplay.forEach(browserId => {
         browserModuleMap.set(
-          browserId,
-          executionJson.executionMapping[browserId]
+          browserId.toString(),
```

Comment:
what is browserId prior to coercing it into a string here?

Reply:
when we use an option `replay-browser=[num]` the value for the option is number, as in a json file, test-execution.file, it writes browserIds as string it should coerce into a string so that they are using the same type to set and get a module map from the set

Reply:
^ @stefanpenner

Reply:
Thank you